README.md

```diff
@@ -121,3 +121,46 @@ You can also pass nativefier flags, and mount additional volumes to provide loca
 ## License
 
 [MIT](LICENSE.md)
+
+## TS: Minimum to release beta
+
+- [x] Move cli to TS
+- [x] Move app to TS
+- [x] Package.json nits: move away from "^A.B.C" to more understandable "A.x
+- [x] Upgrade electron-packager and other deps
+- [x] Fix node_modules badly pruned when copied to app
+- [x] Get rid of Babel / webpack
+- [x] Move from Gulp to simple npm scripts
+- [x] Replace quirky & broken `progress` with basic logging and restore electron-packager logging
+- [x] Replace `async` module + callbacks with real native async/await
+- [x] Cleanup/port old npm tasks/scripts
+- [x] Make exports more idiomatic without default & index.ts crap
+- [x] Get rid of micro-packages easily inlined
+- [x] Cleanup pass on cli/app code to make more modern async TS idiomatic
+- [x] More verbose & normal logging everywhere
+- [x] Disable logging by default in tests & add tasks to enable at any level.
+- [x] Fix icon not showing up on Linux
+- [x] Review .npmignore and published zip. Test installing it.
+- [ ] Stopping to webpack `app` template caused https://github.com/npm/cli/issues/757 .
+      Find a workaround or revert to webpack.
+- [ ] Add tests & typing
+- [ ] Add coverage report and improve coverage
+- [ ] Windows & macOS smoke test
+- [ ] More typing and testing of what's passed to electron-packager
+- [ ] Address TODOs in code
+- [ ] Warn on using electron < 7
+
+## TS: Post-beta/launch improvements
```

Comment:
Possibly pruning of dependencies we use if there's value in doing so - Some of the simple ones can probably be inlined.

Reply:
> Possibly pruning of dependencies we use if there's value in doing so - Some of the simple ones can probably be inlined.

@jiahaog I'm pretty happy with the current state (I marked task `- [x] Get rid of micro-packages easily inlined` as done), but a second pass can come later, sure.